documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc

```diff
@@ -188,13 +189,15 @@ Example output:
 f8dbc7b0a4fcfb1d706e298ac9d0485c2226ce8df7f7596ac77337bd09fbe160
 ----
 
-Writes a 64-byte randomly generated number to the device private key. +
+Writes a 32-byte randomly generated number to the device private key. +
```

Comment:
```suggestion
Writes a 32-byte randomly generated number to the device private key.

```
No need for the trailing `+` here; it actually prevents that warning from displaying as an admonition.